scrowl-io/owl-ui
# [sc-300] - Create Table Component

### Short summary
Create a table component. This component should have multiple variations and props to be applied, such as hover, stripped, and bordered. The table component will not render any rows or columns, the dev will define the content of the table.

---

### Test steps

1. Run `yarn start:storybook`.
2. Checkout the variants on the Storybook and ensure there are no errors in the console log.
